packages/client/hmi-client/src/components/workflow/tera-workflow.vue

```diff
@@ -300,28 +305,34 @@ const teraOperatorRefs = ref();
 async function updateWorkflowName() {
 	const workflowClone = cloneDeep(wf.value.dump());
 	workflowClone.name = newWorkflowName.value;
-	await workflowService.updateWorkflow(workflowClone);
+	await workflowService.saveWorkflow(workflowClone);
 	await useProjects().refresh();
 	isRenamingWorkflow.value = false;
 	wf.value.load(await workflowService.getWorkflow(props.assetId));
 }
 
 // eslint-disable-next-line
 const _saveWorkflow = async () => {
-	await workflowService.updateWorkflow(wf.value.dump());
+	pendingSave = false;
+	await workflowService.saveWorkflow(wf.value.dump());
 	// wf.value.update(updated);
 };
 // eslint-disable-next-line
 const _updateWorkflow = (event: any) => {
-	if (event.data.id !== wf.value.getId()) return;
-
-	// console.log('update workflow', event.data);
-	wf.value.update(event.data as Workflow);
+	if (event.data.id !== wf.value.getId()) {
+		return;
+	}
+	wf.value.update(event.data as Workflow, pendingSave, isDragging);
```

Comment:
If you do not like one line returns, why not reverse the conditional test?
```suggestion
	if (event.data.id === wf.value.getId()) {
		wf.value.update(event.data as Workflow, pendingSave, isDragging);
	}
```